### ISSUE TITLE
Chose Plone 4.3-latest to create a package, but it's pinning to test_plone52.cfg in buildout

### ISSUE THREAD

**Issue by idgserpro** [plone/bobtemplates.plone#383]
I created a package with plonecli, setting do Plone 4.3. Then, ran `plonecli build`. At the end of the buildout, I got:

```
# Required by:
# collective.themesitesetup==2.1.2
zope.app.localpermission = 4.1.0
Picked versions have been written to test_plone52.cfg
```

When it should have been to test_plone43.cfg.

**Comment by MrTango** [comment]
Yes but one can easily change this. I leave this issues open, so that we can improve this a bit by use the version info to change the includes. But you don't need these files, they more there to make thinks easier. With the included tox setup you can always test and play with all versions.

**Comment by idgserpro** [comment]
> Yes but one can easily change this.

I agree. Actually, that's what it's being done. But it's not ideal. The idea of using specific templates is exactly to avoid doing these manual editions which are error prone.

> But you don't need these files, they more there to make thinks easier. 

I'm trying to do what *I think* was the plonecli idea (https://community.plone.org/t/why-do-we-have-plonecli-and-bobtemplates-plone/6217): `plonecli create` and `plonecli build` should give me a default usable instance, in 4.3 or 5.2, without having to mess up with manual edition with configuration files. It's a good approach specially for beginners.
